### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Atteroot**
[DEMO LINK](https://Atteroot.github.io/react_movies-list-fetch-movies/)

### CODE REVIEW

**Review by Andrii1996 — APPROVED** [review]

File: src/components/FindMovie/FindMovie.js [hunk 1]
```diff
@@ -1,55 +1,106 @@
...
+
+        setFoundMovie({
+          title: movie.Title,
+          description: movie.Plot,
+          imgUrl: movie.Poster,
+          imdbUrl: `https://www.imdb.com/title/${movie.imdbId}`,
+          imdbId: movie.imdbID,
+        });
+      });
+  };
```
Comment (danheim) [inline src/components/FindMovie/FindMovie.js:33]: async await

File: src/components/FindMovie/FindMovie.js [hunk 2]
```diff
@@ -1,55 +1,106 @@
...
+};
+
+FindMovie.propTypes = {
+  addMovie: PropTypes.func.isRequired,
+  movies: PropTypes.arrayOf(PropTypes.shape({
+    title: PropTypes.string.isRequired,
+    description: PropTypes.string.isRequired,
+    imgUrl: PropTypes.string.isRequired,
+    imdbUrl: PropTypes.string.isRequired,
+  }).isRequired).isRequired,
```
Comment (danheim) [inline src/components/FindMovie/FindMovie.js:105]: Create shape for movie

**Review by danheim — CHANGES_REQUESTED** [review]
Great start

**Review by Vladimir-Beliy — APPROVED** [review]

**Review by Vladimir-Beliy — CHANGES_REQUESTED** [review]
/

**Review by Vladimir-Beliy — APPROVED** [review]

**Pull request closed without merging**